### PR TITLE
Remove Spinner component button variant

### PR DESCRIPTION
**Pull request by aduth**
**Why**: This is out of sync with current guidance, where buttons which trigger a waiting experience have since been refined within the IdP to display with a pulsing dot animation.

See: LG-3730, LG-3787, https://github.com/18F/identity-idp/pull/4517

Future considerations:

- Consider backporting spinner button implementation from IdP to design system.

### CODE REVIEW

File: docs/_components/spinner.md [hunk 1]
```diff
@@ -26,25 +26,3 @@ Spinners can be helpful to signal to a user that the current process may require
...
-## Using a button click to launch a spinner
-
-We may want to start with a hidden spinner and then remove it with a button click will the page transition resolves. 
-
-{% capture example %}
-<button class="usa-button__lg-invokeSpinner">Hide Button and trigger a spinner</button>
-<div class="lg-spinner lg-spinner--hidden" id="spinner-id">
-  <div>
-    <img
-      src="{{ site.baseurl }}/assets/img/spinner.gif"
```
Comment (zachmargolis) [inline docs/_components/spinner.md:39]: should we remove the `spinner.gif` and `@2x` files from this repo as well?

**Review by zachmargolis — APPROVED** [review]
LGTM

File: docs/_components/spinner.md (hunk 1 above)
Comment (aduth) [reply docs/_components/spinner.md:39]: > should we remove the `spinner.gif` and `@2x` files from this repo as well?

The spinner component itself will continue to exist after this, just not the button variant. We're still using the spinner in at least one place in the IdP (IAL2 Acuant modal overlay loading state).

**Comment by aduth** [comment]
Visual regression is expected for removed content from Spinner page.

**Pull request merged**